task_3/task_3/Program.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace task_3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Random rnd = new Random();
+            int size = 10, k;
+            int[] nums = new int[size];
+            Console.WriteLine("Массив случайных чисел!");
+
+            for (k = 0; k < nums.Length; k++)
```

Comment:
for (k = 0; k < nums.Length; k++)
 
better ` for (int (or var)k = 0; k < nums.Length; k++)`